Source/Bogus/Person.cs

```diff
@@ -55,13 +55,17 @@ public class CardCompany
       /// the Randomizer.Seed global static is ignored and a locally isolated derived seed is used to derive randomness.
       /// However, if the <paramref name="seed"/> parameter is null, then the Randomizer.Seed global static is used to derive randomness.
       /// </param>
-      public Person(string locale = "en", int? seed = null)
+      public Person(string locale = "en", int? seed = null, DateTime? refDate = null)
       {
          this.GetDataSources(locale);
          if( seed.HasValue )
          {
             this.Random = new Randomizer(seed.Value);
          }
+         if( refDate.HasValue )
+         {
+            this.DsDate.LocalSystemClock = () => refDate.Value;
+         }
          this.Populate();
       }
 
```

Comment:
TODO: Need to figure out what to do for the internal constructor 😞
* `internal Person(Randomizer randomizer, string locale = "en")`

Reply:
Having a second ctor as with the Func<DateTime?> as there is for the Date and passing () => localRefDate or FakerHub.Date.LocalSystemClock from the Faker should work I maybe?

Reply:
I pushed a proposal in #492.